logging/google/cloud/logging/_helpers.py

```diff
@@ -46,3 +52,31 @@ def entry_from_resource(resource, client, loggers):
         return ProtobufEntry.from_api_repr(resource, client, loggers)
 
     raise ValueError('Cannot parse log entry resource.')
+
+
+def retrieve_metadata_server(metadata_key):
+    """Retrieve the metadata key in the metadata server.
+
+    See: https://cloud.google.com/compute/docs/storing-retrieving-metadata
+
+    :type metadata_key: str
+    :param metadata_key: Key of the metadata which will form the url.
+
+    :rtype: str
+    :returns: The value of the metadata key returned by the metadata server.
+    """
+    url = METADATA_URL + metadata_key
+
+    try:
+        response = requests.get(url, headers=METADATA_HEADERS)
+
+        if response.status_code == requests.codes.ok:
```

Comment:
I looked at the [raise_for_status](https://github.com/requests/requests/blob/205755834d34a8a6ecf2b0b5b2e9c3e6a7f4e4b6/requests/models.py#L912) method, it only throws exception when status_code is greater than 400. But I think it's safer to only return response when status_code is 200. So I changed it back.